library/Ivoz/Provider/Domain/Service/Administrator/AssertWebPortalAccessible.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Ivoz\Provider\Domain\Service\Administrator;
+
+use Ivoz\Provider\Domain\Model\Administrator\AdministratorInterface;
+use Ivoz\Provider\Domain\Model\WebPortal\WebPortal;
+use Ivoz\Provider\Domain\Model\WebPortal\WebPortalInterface;
+use Ivoz\Provider\Domain\Model\WebPortal\WebPortalRepository;
+
+final class AssertWebPortalAccessible
+{
+    public function __construct(
+        private WebPortalRepository $webPortalRepository
+    ) {
+    }
+
+    public function execute(AdministratorInterface $user, string $domain): void
+    {
+        if ($user->isSuperAdmin()) {
```

Comment:
Does this apply the rule `No se aplican limitaciones si la parte host de la URL está dado de alta como URL de un nivel distinto al que pertenece el administrador que se está logando  (e.g. dominio de brand logándose con /client con credenciales de cliente: no hay limitaciones)`?

Reply:
That wording is a bit confusing. The rule is "no one should see a WebPortal theme owned by a company (brand) they don't belong to". 

So if you are a SuperAdmin no limitation applies to you.